types/wx-js-sdk-dt/index.d.ts

```diff
@@ -0,0 +1,468 @@
+// Type definitions for wx-js-sdk 1.2
+// Project: https://res.wx.qq.com/open/js/jweixin-1.2.0.js
+// Definitions by: Bian Zhongjie <https://github.com/agasbzj>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+export namespace wx {
```

Comment:
'wx' will be the only global variable, and all the functions and classes are in the 'wx' object in js. Sorry about the tests, I'm still learning how to write them. By the way, I have used most of the functions in my projects.